component/application-controller.jsonnet

```
@@ -27,6 +27,7 @@ local objects = [
               '--app-resync',
               std.format('%d', params.resync_seconds),
             ],
+            resources: params.resources.application_controller,
```

I would suggest the following pattern to allow users to remove the default requests/limits if they really want:

```suggestion
            [if params.resources.application_controller != null then 'resources']:
              std.prune(params.resources.application_controller),
```

We only add the section, if the user hasn't set the full key to null in the hierarchy, and prune the provided config to allow users to clear individual requests or limits in the hierarchy.